src/components/cylc/tree/TreeItem.vue

```diff
@@ -32,50 +32,47 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <!-- the node value -->
       <!-- TODO: revisit these values that can be replaced by constants later (and in other components too). -->
       <div :class="getNodeDataClass()" @click="nodeClicked" v-if="node.type === 'cyclepoint'">
-        <CylcObject :id="node.node.id">
-          <task
-           :status="node.node.state"
-           :isHeld="node.node.isHeld"
-           :progress=0
-          />
-        </CylcObject>
+        <task
+          v-cylc-object="node.node.id"
```

Comment:
Passing the ID here, to the directive, avoids the directive code from having to go through the DOM attributes to fetch from the element ID :+1: